src/graph.js

```diff
@@ -1,8 +1,162 @@
 /* eslint-disable */
+class Node {
```

Comment:
Nice idea to encapsulate each graph node as its own class. It does make the implementation of `addEdge` and `removeEdge` a lot less convoluted.